editor/src/core/shared/element-path.ts

```diff
@@ -328,8 +328,15 @@ export function fullDepth(path: ElementPath): number {
   return path.parts.reduce((working, part) => working + part.length, 0)
 }
 
-export function isInsideFocusedComponent(path: ElementPath): boolean {
-  return path.parts.length > 2
+export function isInsideFocusedComponent(
```

Comment:
I think I'd be tempted to put these in a different file as it feels a bit like it's diluting the purpose of this one, but I certainly wouldn't protest if you didn't.

Reply:
Yeah I felt the same way tbh but couldn't think of a better place to put them.